includes/scripts.php

```diff
@@ -234,6 +234,7 @@ function edd_load_admin_scripts( $hook ) {
 		'show_advanced_settings'      => __( 'Show advanced settings', 'easy-digital-downloads' ),
 		'hide_advanced_settings'      => __( 'Hide advanced settings', 'easy-digital-downloads' ),
 		'no_downloads_error'          => __( 'There are no downloads attached to this payment', 'easy-digital-downloads' ),
+		'wait'                        => __( 'Please wait . . .', 'easy-digital-downloads' ),
```

Comment:
We should switch this to `&hellip;` instead of `. . .`

Reply:
Done.